README.md

```diff
@@ -1,53 +1,31 @@
 # Beluga
 
-![CI badge](https://github.com/Ekumen-OS/beluga/actions/workflows/ci_pipeline.yml/badge.svg?event=push)
+[![CI pipeline](https://github.com/Ekumen-OS/beluga/actions/workflows/ci_pipeline.yml/badge.svg?branch=main)](https://github.com/Ekumen-OS/beluga/actions/workflows/ci_pipeline.yml?query=branch:main)
+[![License Apache-2.0](https://img.shields.io/badge/license-Apache--2.0-blue.svg)](LICENSE)
 
 ## Overview
 
-Beluga is an extensible library with a ground-up implementation of the Monte Carlo Localization (MCL) family of estimation algorithms,
-including sensor and motion models.
-It also provides a _drop-in_ replacement for the `amcl` node used in [navigation2](https://github.com/ros-planning/navigation2).
+Beluga is an extensible C++ library with a ground-up implementation of the Monte Carlo Localization (MCL) family of estimation algorithms featuring:
 
-https://github.com/Ekumen-OS/beluga/assets/33042669/df2a5e61-3d97-4350-b171-71bb1a09a6d0
+- A modular design based on orthogonal components that can be merged together using the mixin pattern.
+- Good test coverage to prevent regressions and facilitate code improvements.
+- Automated benchmarks that can be used to validate different configurations.
```

Comment:
Should we say "semi-automated" at this point, since the user is still in the loop?

Reply:
I might be wrong, but the benchmarks are automated AFAIK.

Reply:
The benchmark execution is automated, but the benchmark still needs to be started by the user, right? In my head, fully automated does not require user intervention at all: it starts, runs and generates a report autonomously (e.g. runs in CI for all updates to main / for all new releases).

And maybe related to the distinction, by benchmarks here do you mean the micro-benchmarks that run in tests, or the beluga_benchmark package that generates reports?